src/Compilers/Core/Portable/MetadataReader/MetadataReaderExtensions.cs

```diff
@@ -1,8 +1,10 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
 using System;
+using System.Collections.Generic;
 using System.Collections.Immutable;
 using System.Globalization;
+using System.Linq;
```

Comment:
Additional usings in this file a no longer necessary, consider reverting the file to its original state.